db/migrate/20161011213434_acts_as_votable_migration.rb

```diff
@@ -0,0 +1,27 @@
+class ActsAsVotableMigration < ActiveRecord::Migration
+  def self.up
+    create_table :votes do |t|
```

Comment:
Since we're not using the `acts_as_votable` gem in any of our models, we should probably create migrations to remove this table (and the `voting_sessions` table) from the database schema.